internal/tss/utils/types.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import (
+	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/wallet-backend/internal/tss"
+)
+
+type TransactionService interface {
```

Comment:
Let's keep the interface together with the struct definition, it's easier to reference one another and they're both types.

Reply:
Fair enough